.github/workflows/flutter_ci.yml

```
@@ -0,0 +1,70 @@
+
+      - name: Get dependencies
+        run: flutter pub get
+
+      - name: Run analyzer
+        run: flutter analyze --fatal-infos
+
+      - name: Run tests
+        run: flutter test --coverage
+
+      - name: Build for Linux (sanity check)
+        if: matrix.os == 'ubuntu-latest'
```

actionlint (1.7.7)

property "os" is not defined in object type {}

(expression)